nixos/modules/services/security/step-ca.nix

```diff
@@ -108,6 +108,9 @@ in
           Environment = "HOME=%S/step-ca";
           WorkingDirectory = ""; # override upstream
           ReadWriteDirectories = ""; # override upstream
+          # systemd renamed the above to `ReadWritePaths`, but upstream
+          # still uses `ReadWriteDirectories`, and it is unclear if the
+          # override works when the alias is used.
```

Comment:
```suggestion
          ReadWritePaths = ""; # override upstream
```

systemd handles aliases properly.

This works as expected.

You can confirm this interactively in the NixOS VM test using `systemctl cat step-ca` and `systemctl show step-ca --all`.

Reply:
Thanks, I'll use that then!